tools/osv-scanner/main.go

```diff
@@ -1,38 +1,22 @@
 package main
 
 import (
+	"bufio"
 	"bytes"
 	"errors"
-	"flag"
-	"fmt"
 	"log"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
 
+	"github.com/g-rath/osv-detector/pkg/lockfile"
+	"github.com/urfave/cli/v2"
+
 	"github.com/google/osv.dev/tools/osv-scanner/internal/osv"
 	"github.com/google/osv.dev/tools/osv-scanner/internal/sbom"
 )
 
-func usage() {
-	fmt.Fprintf(flag.CommandLine.Output(), "%s [flags] dir1 dir2...\n", os.Args[0])
-	flag.PrintDefaults()
-}
-
-func scan(query *osv.BatchedQuery, arg string) error {
-	info, err := os.Stat(arg)
-	if err != nil {
-		return nil
-	}
-
-	if info.IsDir() {
-		return scanDir(query, arg)
-	}
-
-	return scanFile(query, arg)
-}
-
 func scanDir(query *osv.BatchedQuery, dir string) error {
 	log.Printf("Scanning dir %s\n", dir)
```

Comment:
Can you add a todo here to recursively detect and scan all files too, not just nested git repos?

Reply:
Now scans all files in 030f8da